build/charts/antrea/values.yaml

```diff
@@ -28,8 +28,14 @@ transportInterface: ""
 # -- Network CIDRs of the interface on Node which is used for tunneling or
 # routing the traffic across Nodes.
 transportInterfaceCIDRs: []
-# -- Names of the interfaces on Nodes that are used to forward multicast traffic.
-multicastInterfaces: []
+
+multicast:
+  # -- Names of the interfaces on Nodes that are used to forward multicast traffic.
+  multicastInterfaces: []
+  # -- The interval for antrea-agent to send IGMP query to pods, time uint is second.
+  # Defaults to 125 seconds.
```

Comment:
you can remove this IMO, the default is already used to configure the value below

Reply:
done